core/sqf/sqenvcom.sh

```diff
@@ -95,17 +95,20 @@ cpucnt=$(grep processor /proc/cpuinfo | wc -l)
 #     no number means unlimited, and will swamp the system
 export MAKEFLAGS="-j$cpucnt"
 
+# For now set up the TOOLSDIR, it may be overwritten later when the
+# .trafodion configuration file is loaded.
 if [[ -n "$CLUSTERNAME" ]]; then
-  export MY_ROOT=/opt/hp
-  export TOOLSDIR=${TOOLSDIR:-/home/tools}
+  if [ -z "$TOOLSDIR" ]; then
+    export TOOLSDIR=${TOOLSDIR:-/home/tools}
+  fi
```

Comment:
I think what Steve is saying that the old syntax

export TOOLSDIR=${TOOLSDIR:-/home/tools}

already does that, it does nothing if TOOLSDIR is already set.